src/commandSocket.cpp

```diff
@@ -629,7 +629,18 @@ void CommandSocket::sendPrintTimeMaterialEstimates()
     logDebug("Sending print time and material estimates.\n");
     auto message = std::make_shared<cura::proto::PrintTimeMaterialEstimates>();
 
-    message->set_time(FffProcessor::getInstance()->getTotalPrintTime());
+    std::unordered_map<PrintFeatureType, double> time_estimates = FffProcessor::getInstance()->getTotalPrintTimes();
```

Comment:
Because enums evaluate to integer types it's better to just make this a vector.
Do make sure the enum constants are the right integer values by defining them.

E.g.:
````
enum class PrintFeature
{
   INFILL = 0,
   SKIN = 1,
    etc.
}
time_estimates[PrintFeature::INFILL] = ...;
````